### PR TITLE
Bankaccounts

### PR DESCRIPTION
**Pull request by ivankraev**
## Creating Dashboard for bank accounts entity according to figma design.
- There are still some things to be done for the pr to look complete and i will improve it.

pull3
pull1
pull2

### CODE REVIEW

File: dasdsa [hunk 1]
```diff
@@ -0,0 +1,67 @@
+
+                   SSUUMMMMAARRYY OOFF LLEESSSS CCOOMMMMAANNDDSS
```
Comment (quantum-grit) [inline dasdsa:2]: It looks like this file is not part of the code

File: src/common/hooks/bankaccounts.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { endpoints } from 'common/api-endpoints'
+import { BankAccountResponse } from 'gql/bankaccounts'
```
Comment (quantum-grit) [inline src/common/hooks/bankaccounts.ts:2]: the automated build fails on this line, could you check if you have committed all needed files

File: src/components/bankaccounts/BankAccountsBottomAppBar.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+import Tooltip from '@mui/material/Tooltip'
+import { useTranslation } from 'next-i18next'
+import Toolbar from '@mui/material/Toolbar'
+import PrintIcon from '@mui/icons-material/Print'
+import SaveIcon from '@mui/icons-material/Save'
+import { observer } from 'mobx-react'
+import ShareIcon from '@mui/icons-material/Share'
+import EventNoteIcon from '@mui/icons-material/EventNote'
+import { AlertStore } from 'stores/AlertStore'
+import { ModalStore } from 'stores/cars/ModalStore'
```
Comment (quantum-grit) [inline src/components/bankaccounts/BankAccountsBottomAppBar.tsx:13]: please remove car related files

**Review by quantum-grit — CHANGES_REQUESTED** [review]
great job overall, looks nice UI, please remove car related files and fix the build

File: src/common/hooks/bankaccounts.ts (hunk 2 above)
Comment (ivankraev) [reply src/common/hooks/bankaccounts.ts:2]: I checked it, everything seems correct. Not sure why its not happy with this import tbh.

File: src/common/routes.ts [hunk 4]
```diff
@@ -53,8 +53,13 @@ export const routes = {
...
     supporters: '/admin/supporters',
   },
-  dev: {
-    openData: '/open-data',
-    apiDocs: '/docs',
+  bankaccounts: {
+    index: '/bankaccounts',
+    edit: (id: string) => `/tasks/edit/${id}`,
+    add: '/bankaccounts/add',
+    dev: {
```
[the remark this answers is not in the document]
Comment (ivankraev) [reply src/common/routes.ts]: fixed

**Review by kachar — APPROVED** [review]

**Pull request merged**